Week1/homework/js-exercises/arrayAnimals.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+let items =[];
+console.log("this is an empty array"];
+console.log(items);
+let animals=["bird","fish", "cat"];
+console.log(animals);
+animals.push("Piglets");
```

Comment:
Nice seeing that you discovered the `push` method early on, it's an often used method for handling data within an array!